servers/rendering/renderer_canvas_cull.cpp

```diff
@@ -248,6 +245,22 @@ void RendererCanvasCull::_cull_canvas_item(Item *p_canvas_item, const Transform2
 	}
 
 	Transform2D xform = ci->xform;
+	Point2 repeat_size = p_repeat_size;
+	int repeat_times = p_repeat_times;
+
+	if (ci->repeat_source) {
+		repeat_size = ci->repeat_size;
+		repeat_times = ci->repeat_times;
+	} else {
+		ci->repeat_size = repeat_size;
+		ci->repeat_times = repeat_times;
+
+		if (repeat_size.x || repeat_size.y) {
+			rect.size += repeat_size * repeat_times / ci->xform.get_scale();
+			rect.position -= repeat_size * (repeat_times / 2);
```

Comment:
Is this intended to do integer division here? Just making sure

Reply:
Yes. I wasn't sure if there was a more performant way to truncate the result.

Reply:
Then it's good